src/launchWizard/launchWizard.ts

```diff
@@ -280,10 +282,14 @@ class LaunchWizard {
   getWebViewContent() {
     const scriptUri = vscode.Uri.parse(
       this.ctx.asAbsolutePath('./src/launchWizard/launchWizard.js')
-    ).with({ scheme: 'vscode-resource' })
+    )
     const styleUri = vscode.Uri.parse(
       this.ctx.asAbsolutePath('./src/launchWizard/styles.css')
-    ).with({ scheme: 'vscode-resource' })
+    )
+    const scriptData = fs.readFileSync(scriptUri.fsPath)
+    const styleData = fs.readFileSync(styleUri.fsPath)
+
+    // const styleUri = this.getPanel().webview.asWebviewUri(stylePath)
```

Comment:
Commented out code. I'd say delete or add a comment to the line why you want it retained. 
Eg. "// Todo: keep for now. Still thinking about this."